### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by dhpalmer** [DessimozLab/read2tree#19]
If you could please help I'd appreciate it.

When running the following:

`conda install -c bioconda  read2tree`

I get this output and error:

```
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: \ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                                                                                                    

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.17=0
  - feature:|@/linux-64::__glibc==2.17=0

Your installed version is: 2.17
```

**Comment by sinamajidian** [comment]
Hi @dhpalmer 

Thanks for using read2tree. 

Could you please provide us more information about your system and python/conda version and the full command line you used? 
We would suggest to create a new environment as described in [the readme](https://github.com/DessimozLab/read2tree#1-installation-using-conda) with python 3.10. 

It seems that such an error may arise if you have incompatible python version as [described here](https://stackoverflow.com/questions/71678461/conda-unsatisfiableerror-with-linux64-and-glibc) or outdated conda version 
mentioned [here](https://stackoverflow.com/a/67156339/9648531).

Regards,
Sina

**Comment by bfauskee** [comment]
Hello,

I am getting the same issue on an AWS system:

conda install -c bioconda  read2tree 
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: - 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                            

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.35=0
  - feature:|@/linux-64::__glibc==2.35=0

Your installed version is: 2.35

I have conda 23.3.1 and am in an environment with python 3.10.11

**Comment by sinamajidian** [comment]
Dear @bfauskee  

Thanks for reporting the issue in details. I was able to reproduce it and we will investigate the issue in.

In the meanwhile, I would suggest to use python 3.10.8 as I just tested with a fresh environment and read2tree works well.  (It may take a while for conda to resolve the environment and install read2tree).

I would appreciate it if you could inform us about the result.

Regards,
Sina

**Comment by bfauskee** [comment]
Yep this worked for me. Seems to be running well on python 3.10.8. Thanks!

**Comment by CJJ8848** [comment]
Hi @sinamajidian,

I also have a trouble with the conda install. I built up a fresh env with python=3.10.8 and conda 23.3.1 in a Linux system. Here it is:

conda create -n r2t3.10.8 python=3.10.8 
conda activate r2t3.10.8
(r2t3.10.8) conda install -c bioconda  read2tree 
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: | 
/ 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                              

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versionsThe following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.17=0
  - feature:|@/linux-64::__glibc==2.17=0

Your installed version is: 2.17





I also tried to build read2tree from source in python 3.8 and succeded. I also succeded to run the reference with 'read2tree --standalone_path marker_genes/ --output_path output --reference' Then when I tried to run multispecies mode with the command 'read2tree --standalone_path marker_genes/ --output_path output --reads species1_R1.fastq species2_R2.fastq', it throwed an error 'ValueError: fetch called on bamfile without index' in Mapper.py 499 line. I am confused about this and have kept debugging for three days. I guess there should be some conficts happened after installing from source as well. I would aprreciate it if you could help me out.
